django/db/models/fields/tuple_lookups.py

```diff
@@ -34,52 +38,68 @@ def check_rhs_length_equals_lhs_length(self):
                 f"must have {len_lhs} elements"
             )
 
-    def as_sql(self, compiler, connection):
-        # e.g.: (a, b, c) == (x, y, z) as SQL:
-        # WHERE (a, b, c) = (x, y, z)
-        vals = [
+    def get_prep_lhs(self):
+        if isinstance(self.lhs, (tuple, list)):
+            return Tuple(*self.lhs)
+        else:
+            return super().get_prep_lhs()
+
+    def process_lhs(self, compiler, connection, lhs=None):
+        sql, params = super(BuiltinLookup, self).process_lhs(compiler, connection, lhs)
+        if not sql.startswith("("):
+            sql = f"({sql})"
+        return sql, params
```

Comment:
I'm not confident this is the best way to check if something needs wrapping with parenthesis 
I'm not sure if something like `(complex expression 1), expression 2, (complex expression 3)` could be possible :thinking:

Reply:
Yes SQLite is intelligent enough to compile this correctly, but I've never seen this syntax before so I would like to avoid it if possible.

That said, it _does_ look like a hack. I'm gonna give it some thought.

Reply:
I added a small adjustment that looks a little less hacky, please check!

Reply:
I prefer it thank you